documentation/modules/error-messages.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="error-messages_{context}"]
+= Error messages
+
+This section describes error messages and how to resolve them.
+
+.warm import retry limit reached
+
+The `warm import retry limit reached` error message is displayed during a warm migration if a VMware virtual machine (VM) has reached the maximum number (28) of changed block tracking (CBT) snapshots during the precopy stage. You must delete some of the CBT snapshots from the VM and restart the migration plan.
```

Comment:
@sbeskin-redhat 
```suggestion
Displayed during a warm migration if a VMware virtual machine (VM) has reached the maximum number (28) of changed block tracking (CBT) snapshots during the precopy stage. You must delete some of the CBT snapshots from the VM and restart the migration plan.
```
Maybe change "some" to indicate that you need to delete enough such that the total is less than or equal to the maximum.

Reply:
Will open a separate bug for editing the module's content

Reply:
@sbeskin-redhat Sounds good.